### PR TITLE
OT153-48 Shared title component with uppercase pipe

### PR DESCRIPTION
**Pull request by NLuna19**
## JIRA
[OT153-48](https://alkemy-labs.atlassian.net/browse/OT153-48)

## Short description

Created a shared component that receives a text and a background image, you can also customize the color of the text.
The component can receives the following variables:
- **title**: string;
- **pathBackground**: string;
- **colorBackground**: string;
- **colorText**: string;

## Screenshot or Video

image
image

### Sanity Check Steps

- [x] All console logs have been removed.
- [x] This change was tested on my localhost.
- [ ] This change depends for another ticket.
- [x] All linked JIRA acceptance criteria are satisfied.
- [ ] Does this change require environment variables.

### CODE REVIEW

File: src/app/shared/components/upper-title/upper-title.component.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<div>
+    <div [style]="{
```
Comment (RicardoMaldonado93) [inline src/app/shared/components/upper-title/upper-title.component.html:2]: Agregaría una propiedad extra para cambiar el color de fondo

File: src/app/shared/components/upper-title/upper-title.component.scss [hunk 2]
```diff
@@ -0,0 +1,16 @@
+$colorText: black;
+.text-title{
+    font-family: '-apple-system', 'BlinkMacSystemFont', 'Segoe UI', 'Roboto', 'Helvetica', 'Arial', 'sans-serif', 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol';
```
Comment (RicardoMaldonado93) [inline src/app/shared/components/upper-title/upper-title.component.scss:3]: Me gustaría que esté en una variable al igual que el color

File: src/app/shared/components/upper-title/upper-title.component.scss [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    font-size: 2rem;
+    text-align: center;
+    color: $colorText;
+}
+
+.container-title{
+    padding: 1px;
+    margin: 0;
+    border-radius: 5px;
+    background-color: rgb(252, 96, 85);
```
Comment (RicardoMaldonado93) [inline src/app/shared/components/upper-title/upper-title.component.scss:14]: Debería estar en una variable al igual que el color y fuente

File: src/app/shared/components/upper-title/upper-title.component.ts [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+@Component({
+  selector: 'app-upper-title',
+  templateUrl: './upper-title.component.html',
+  styleUrls: ['./upper-title.component.scss']
+})
+export class UpperTitleComponent implements OnInit {
+  @Input()title!: string;
+  @Input()pathBackground!: string;
+  @Input()colorText!: string;
+
```
Comment (RicardoMaldonado93) [inline src/app/shared/components/upper-title/upper-title.component.ts:12]: Agregaría un backgroundColor

File: src/app/app.module.ts [hunk 5]
```diff
@@ -14,6 +15,7 @@ import { AppComponent } from './app.component';
     BrowserModule,
     CoreModule,
     FeaturesModule,
+    SharedModule
```
Comment (RicardoMaldonado93) [inline src/app/app.module.ts:18]: El SharedModule está importado dentro del FeatureModule, hay que dejar este y borrar el otro, o borrar este y dejar el otro.

**Review by RicardoMaldonado93 — COMMENTED** [review]
@NLuna19 Please fix the observation

**Comment by NLuna19** [comment]
All fixed

**Review by RicardoMaldonado93 — APPROVED** [review]
Good job!

**Pull request merged**